README.md

```diff
@@ -49,26 +39,33 @@ Presenter
 @InjectViewState
```

Comment:
We can remove @InjectVIewState from the sample project.

Reply:
Done